src/vermont.cc

```diff
@@ -131,17 +131,17 @@ static int
 parse_log_level (const char *arg, int mask)
```

Comment:
Do we still need the `mask` function parameter? It seems that we can just directly return `LOG_UPTO()`

Reply:
Yes, that would make a good tidy up. I've removed it.